Stable VMR builds are broken

> D:\a\_work\1\s\.dotnet\sdk\11.0.100-preview.1.26078.121\NuGet.Build.Tasks.Pack.targets(222,5): error NU5104: Warning As Error: A stable release of a package should not have a prerelease dependency. Either modify the version spec of dependency "Microsoft.DotNet.Build.Tasks.Templating [11.0.0-beta.26103.104, )" or update the version field in the nuspec. [D:\a\_work\1\s\src\runtime\src\coreclr\.nuget\Microsoft.NET.Sdk.IL\Microsoft.NET.Sdk.IL.proj]

--- a/src/coreclr/.nuget/Microsoft.NET.Sdk.IL/Microsoft.NET.Sdk.IL.proj
+++ b/src/coreclr/.nuget/Microsoft.NET.Sdk.IL/Microsoft.NET.Sdk.IL.proj
@@ -1,4 +1,5 @@
 <Project Sdk="Microsoft.Build.NoTargets">
+
   <PropertyGroup>
     <IsPackable>true</IsPackable>
     <EnablePackageValidation>false</EnablePackageValidation>
@@ -8,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Templating" Version="$(MicrosoftDotNetBuildTasksTemplatingVersion)" />
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Templating" Version="$(MicrosoftDotNetBuildTasksTemplatingVersion)" PrivateAssets="all" />
   </ItemGroup>
 
   <ItemGroup>
@@ -24,7 +25,6 @@
   <Target Name="ReplaceTemplateParametersInILTargetsTemplate"
           Inputs="$(MSBuildThisFileFullPath);$(ILTargetsTemplateFile)"
           Outputs="$(ILTargetsOutputFile)">
-
     <GenerateFileFromTemplate TemplateFile="$(ILTargetsTemplateFile)"
                               OutputPath="$(ILTargetsOutputFile)"
                               Properties="IlAsmVersion=$(Version)" />

